library/lib.sh

```diff
@@ -257,35 +255,69 @@ lsrConvertToCollection() {
 
 lsrGetManagedNodes() {
     local guests_yml=$1
-    yq -r ". | keys[] | select(test(\"managed*\"))" "$guests_yml" | sort
+    sed --quiet --regexp-extended 's/(^managed.*):/\1/p' "$guests_yml" | sort
```

Comment:
```suggestion
    sed --quiet --regexp-extended 's/(^managed.*):.*$/\1/p' "$guests_yml" | sort
```
use `.*$` to replace the entire line with the matched expression